project.ncl

```diff
@@ -73,5 +73,11 @@ let import_nix = organist.nix.import_nix in
     },
 
   flake.checks = import "tests/main.ncl",
+
+  files.".gitignore".materialisation_method = 'Copy,
+  files.".gitignore".content = m%"
+    /examples/*/result
+    /result
+  "%,
```

Comment:
```suggestion
    /result

  "%,
```

We should have a newline at the end there. We could also have a separate field like `add_newline` that would by default add a newline at the end of file.